internal/apis/certmanager/validation/certificaterequest_test.go

```diff
@@ -610,8 +610,10 @@ func TestValidateCertificateRequest(t *testing.T) {
 					Usages:    []cminternal.KeyUsage{cminternal.UsageKeyEncipherment, cminternal.UsageDigitalSignature},
 				},
 			},
-			a:     someAdmissionRequest,
-			wantE: []*field.Error{},
+			a: someAdmissionRequest,
+			wantE: []*field.Error{
+				field.Invalid(fldPath.Child("request"), nil, "csr is invalid: the KeyUsages [] do not match the expected KeyUsages [ 'digital signature', 'key encipherment' ]"),
```

Comment:
This check was missing: in case there is an empty set of KeyUsages encoded in the CSR blob, the validation webhook should detect that this blob does not match the KeyUsages specified in the CertificateRequest Spec.